tests/generic_config_updater/test_dynamic_acl.py

```diff
@@ -0,0 +1,755 @@
+    dynamic_acl_remove_third_drop_rule(rand_selected_dut)
+    dynamic_acl_verify_packets(setup,
+                               ptfadapter,
+                               packets=generate_packets(setup, DST_IP_BLOCKED, DST_IPV6_BLOCKED),
+                               packets_dropped=False,
+                               src_port=setup["scale_port_indices"][2])
+
+
+def test_gcu_acl_forward_rule_priority_respected(rand_selected_dut, ptfadapter, setup, dynamic_acl_create_table):
+    """Test that forward rules and drop rules can be created at the same time, with the forward rules having
+    higher priority than drop.  Then, perform a traffic test to confirm that packets that match both the forward
+    and drop rules are correctly forwarded, as the forwarding rules have higher priority"""
+
+    dynamic_acl_create_forward_rules(rand_selected_dut)
+    dynamic_acl_create_secondary_drop_rule(rand_selected_dut, setup)
+    dynamic_acl_verify_packets(setup, ptfadapter, packets=generate_packets(setup), packets_dropped=False)
+    dynamic_acl_verify_packets(setup, ptfadapter,
+                               packets=generate_packets(setup, DST_IP_BLOCKED, DST_IPV6_BLOCKED),
+                               packets_dropped=True)
+
+
+def test_gcu_acl_forward_rule_replacement(rand_selected_dut, ptfadapter, setup, dynamic_acl_create_table):
+    """Test that forward rules can be created, and then afterwards can have their match pattern updated to a new value.
+    Confirm that packets sent that match this new value are correctly forwarded, and that packets that are sent that
+    match the old, replaced value are correctly dropped."""
+
+    dynamic_acl_create_forward_rules(rand_selected_dut)
+    dynamic_acl_create_secondary_drop_rule(rand_selected_dut, setup)
+    dynamic_acl_replace_rules(rand_selected_dut)
+    dynamic_acl_verify_packets(setup,
+                               ptfadapter,
+                               packets=generate_packets(setup,
+                                                        DST_IP_FORWARDED_REPLACEMENT,
+                                                        DST_IPV6_FORWARDED_REPLACEMENT),
+                               packets_dropped=False)
+    dynamic_acl_verify_packets(setup, ptfadapter, packets=generate_packets(setup), packets_dropped=True)
+
+
+def test_gcu_acl_forward_rule_removal(rand_selected_dut, ptfadapter, setup, dynamic_acl_create_table):
+    """Test that if a forward rule is created, and then removed, that packets associated with that rule are properly
+    no longer forwarded"""
+    dynamic_acl_create_forward_rules(rand_selected_dut)
+    dynamic_acl_create_secondary_drop_rule(rand_selected_dut, setup)
+    dynamic_acl_remove_ipv4_forward_rule(rand_selected_dut)
+    packets = generate_packets(setup)
+    # generate_packets generates packets for both ipv4 and ipv6, but we only want to test that ipv4 is dropped
+    packets.pop('IPV6')
```

Comment:
Is there a particular reason for not testing IPv4 forwarding rule removal? We need to cover both IPv4 and IPv6 removal.